### DISCUSSION TITLE
Since FritzOS 8 problem with Tasmota Webinterface

### DISCUSSION THREAD

**Discussion by kugelkopf123** [arendst/Tasmota#22651] [Support request] [answered]
Good evening,
I have noticed the following.
I recently updated my FritzBox 7530AX to the latest FritzOS 8.0.
Since then I have had problems accessing the web interface of my numerous Tasmota devices.
These include Shelly1PM, Shelly2.5, Shelly2PM, ShellyPlugS, etc. 
All devices are flashed with Tasmota and all have a different software version. From v12 to latest. 
The error does not occur permanently but occasionally. 
The web browser used does not play a role. The error occurs both with a Macbook Pro (Safari, Chrome) and with an iPhone 13.

If the web interface loads, it loads terribly slowly or completely normally.

All Shellys have an IP assigned in the Fritzbox. 
All devices can still be pinged with a very good ping of 3-14ms, even if the web interface is not accessible. 
MQTT works without any problems. It is only the web interface that no longer is accessible.
I have already tried several restarts of all devices, resetting the Fritzbox to factory settings and reconfiguring it. 
Nothing has helped so far.

Since I also log the temperatures of the Shellys, it is also very easy to recognize when they are not accessible. 
Normally, the temperature curve is very flat and they only get a little warmer briefly when the shutters were opend or one or other socket is switched on. But since the update, you can see that the devices get warm without any action. This is usually also the time when the web interface is not accessible. 
Here are two pictures of the logging. One from October and a  current one. 
<img width="875" alt="Bildschirmfoto 2024-12-14 um 23 23 59" src="https://github.com/user-attachments/assets/0c70d3a0-67b2-4d9a-a35a-276175d10bdf" />
<img width="872" alt="Bildschirmfoto 2024-12-14 um 23 21 36" src="https://github.com/user-attachments/assets/5ba4ad94-9ef2-4020-a0b7-a8491c086a3e" />

These problems do not occur with other devices in the network.
There are still two water meters with Ai on the Edge. These ESP32cam are always accessible without any problems.

My hope is that perhaps there are more people who have these problems and have perhaps found a solution in the meantime.
Perhaps it is also related to Tasmota. Because it only shows up on these devices.

**Comment by Jason2866** [comment]
I would try to send out a different SSSID for the IoT devices where all the wifi 6/7 and other modern fancy features are turned off. espressif wifi stack (closed source) is known not to work perfectly with WiFi networks using modern features.

**Comment by f-io** [comment]
You could try this: download.avm.de/inhaus/Smart24P1NL1/7530AX/FRITZ.Box_7530_AX-08.00-117610-Inhaus.image

**Comment by kugelkopf123** [comment]
Thank you very much for your answers. 
I have also contacted AVM support. Before I install an in-house beta firmware, I will first see what they say.
Creating another 2.4ghz network just for the Tasmota devices is not a solution for me.

I may have found the problem. As soon as I can verify this, I will get back to you.

**Comment by spex3142** [comment]
I have also similar problems with tasmota devices since 8.0 What I found was the following: If you activate packet trace on the fritz box the web gui of the tasmota devices work immediately. If you stop trace they are inaccessible again.

**Reply by sfromis** [reply]
That sounds like a Fritzbox issue then....

**Reply by spex3142** [reply]
Yeah, thats also my opinion. The tasmota devices itself are always connected in wlan (as fritzbox shows) and even when not accessible in web gui they work using mqtt. Will open a ticket at AVM.

**Comment by treitlis** [comment, marked as answer]
If anyone is still looking for a solution, AVM support suggested to disable Packet acceleration in the router. The moment I disabled this setting, all devices were accessible through the WebUI. So maybe this is the solution.

**Reply by spex3142** [reply]
I came to the same result with AVM support. This setting solved it for me:
<img width="393" alt="image" src="https://github.com/user-attachments/assets/387c3e44-771f-4947-adc4-199f9974d9bd" />
Problem is after reboot the setting is default again....
